.golangci.yml

```diff
@@ -1,6 +1,4 @@
 run:
-  skip-dirs:
-    - test-cmds
```

Comment:
As asked elsewhere -- is this intentional?

Reply:
updated same as other ask https://github.com/kolide/krypto/pull/39#discussion_r1568929465